update ja_jp.json

karakufire opened the pull request.

3TUSK commented:
@TheLMiffy1111 I hope you can see this message, may I use your translation as reference in Simplified Chinese translation? I appreciate for it. (my computer seems unable to type Chinese, sorry for that.)

TheLMiffy1111 commented:
yes

3TUSK commented:
Thanks very much!

karakufire committed:
update ja_jp.json

 - remove copper ore/block/ingot translation
 - move copper coin/dust/gear/nugget/plate translation into same lines with en_us.json
 - translate untranslated strings
